### ISSUE TITLE
doesnt work

### ISSUE THREAD

**Issue by Am0stafa** [prescottprue/react-redux-firebase#1193]
**Do you want to request a *feature* or report a *bug*?**  

(If this is a *usage question*, please **do not post it here**—post it on [gitter](https://gitter.im/redux-firebase/Lobby). If this is not a “feature” or a “bug”, or the phrase “How do I...?” applies, then it's probably a usage question.)

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [codesandbox](https://codesandbox.io/) or similar.**

**What is the expected behavior?**

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**

**Comment by Saturate** [comment]
On React 18 this is indeed not working.

**Comment by liquidvisual** [comment]
This library doesn't work with React 18? Bummer.

```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: vite-project@0.0.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.3.0 || ^17.0.0" from react-redux-firebase@3.11.0
npm ERR! node_modules/react-redux-firebase
npm ERR!   react-redux-firebase@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
```

**Comment by Saturate** [comment]
> This library doesn't work with React 18? Bummer.
> 
> ```shell
> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR! 
> npm ERR! While resolving: vite-project@0.0.0
> npm ERR! Found: react@18.2.0
> npm ERR! node_modules/react
> npm ERR!   react@"^18.2.0" from the root project
> npm ERR! 
> npm ERR! Could not resolve dependency:
> npm ERR! peer react@"^16.3.0 || ^17.0.0" from react-redux-firebase@3.11.0
> npm ERR! node_modules/react-redux-firebase
> npm ERR!   react-redux-firebase@"*" from the root project
> npm ERR! 
> npm ERR! Fix the upstream dependency conflict, or retry
> ```


Nope, tried to update dependencies for a older project, and it's indeed stuck on react 17.
